xbmc/InfoScanner.h

```diff
@@ -20,14 +20,14 @@ class CInfoScanner
   /*!
     \brief Return values from the information lookup functions.
    */
-  enum INFO_RET
+  enum class InfoRet
   {
-    INFO_CANCELLED,
-    INFO_ERROR,
-    INFO_NOT_NEEDED,
-    INFO_HAVE_ALREADY,
-    INFO_NOT_FOUND,
-    INFO_ADDED
+    CANCELLED,
+    ERROR,
```

Comment:
This will not compile on Windows, because it comes with a macro named `ERROR`.

Reply:
lovely.